bundles/model/org.eclipse.smarthome.model.thing/src/org/eclipse/smarthome/model/thing/internal/util/BundleNameResolver.java

```diff
@@ -0,0 +1,6 @@
+package org.eclipse.smarthome.model.thing.internal.util;
+
+public interface BundleNameResolver {
```

Comment:
Why do you need this additional interface? Wouldn't a making the `getBundleName(...)` method protected be sufficient, so that it can be overridden in the test-case by like this?

```java
new BundleNameResolver() {
    @Override
    protected String getBundleName(...) {
        // ...
    }
}
```

Reply:
done. I didn't like the solution either.